librz/core/cmd.c

```diff
@@ -6553,6 +6553,7 @@ RZ_API char *rz_core_cmd_str(RzCore *core, const char *cmd) {
 	rz_cons_push();
 	if (rz_core_cmd(core, cmd, 0) == -1) {
 		//eprintf ("Invalid command: %s\n", cmd);
+		rz_cons_pop();
```

Comment:
What about extracting this change into the separate PR and merge it right away?

Reply:
yeah that was the plan i think :D